gradle.properties

```diff
@@ -1,17 +1,17 @@
-projectVersion=2.3.4-SNAPSHOT
-micronautVersion=2.3.3
-micronautDocsVersion=1.0.25
+projectVersion=3.0.0-SNAPSHOT
```

Comment:
@ilopmar I'm ready to merge this PR however I have changed the project version to 3.0.0 since I'm going to work on backward non compatible changes.  Is it OK this way? Plus It won't be a snapshot rather then it will be M1 so it is OK to set the version to 3.0.0-M1 for example?

Reply:
It's ok to set it to `3.0.0-SNAPSHOT` because as you said it's a new major version that is not compatible. You don't need to set it to `3.0.0-M1` version. You can do that when doing a release from GitHub Releases page. We always keep snapshot versions in the properties.